pkg/ebpf/ksyms.go

```diff
@@ -23,26 +25,19 @@ func VerifyKernelFuncs(path string, requiredKernelFuncs []string) ([]string, err
 	defer f.Close()
 
 	scanner := bufio.NewScanner(f)
-	scanner.Split(bufio.ScanLines)
-
-	for scanner.Scan() {
-		fields := strings.Fields(scanner.Text())
-		if len(fields) < 3 {
-			continue
-		}
-
-		name := fields[2]
-		if _, ok := found[name]; ok {
-			found[name] = true
+	scanner.Split(bufio.ScanWords)
+	for scanner.Scan() && len(missing) > 0 {
+		if i := missing.Search(scanner.Bytes()); i < len(missing) {
+			missing[0], missing[i] = missing[i], missing[0]
+			missing = missing[1:]
+			sort.Sort(missing)
```

Comment:
Why not do something like 
```go
missing = append(missing[:i], missing[i+1:]...)
```

Reply:
I think that can work too. It allocates, but we don't care about it since it's not in the hot path, and it's more clear so I'll update it.

Reply:
There is also this: https://github.com/golang/go/wiki/SliceTricks#filtering-without-allocating